src/main/java/Deadlines.java

```diff
@@ -0,0 +1,16 @@
+import java.time.LocalDate;
+import java.time.format.DateTimeFormatter;
+
+public class Deadlines extends Task {
+    LocalDate deadLine;
+
+    public Deadlines(boolean isDone, String eventName, String deadLine) {
+        super(isDone, eventName, "D");
+        this.deadLine = LocalDate.parse(deadLine);
+    }
+
+    @Override
+    public String toString() {
+        return "[D] " + super.toString() + "(by: " + deadLine.format(DateTimeFormatter.ofPattern("MMM d yyyy")) + ")";
```

Comment:
Perhaps you could try having this over two lines?

Quoting the java coding standards:
> Try to keep line length shorter than 110 characters (soft limit). But it is OK to exceed the limit slightly (hard limit: 120 chars).

I think the line currently is 110+ characters, but it may improve readability slightly if placed over two lines.